install.sh

```diff
@@ -1,974 +0,0 @@
-#!/usr/bin/env bash
-
-# This is a script to quickly install chaos-mesh.
-# This script will check if docker and kubernetes are installed. If local mode is set and kubernetes is not installed,
-# it will use kind or minikube to install the kubernetes cluster according to the configuration.
-# Finally, when all dependencies are installed, chaos-mesh will be installed using helm.
-
-usage() {
-    cat << EOF
-This script is used to install chaos-mesh.
-Before running this script, please ensure that:
-* have installed docker if you run chaos-mesh in local.
-* have installed Kubernetes if you run chaos-mesh in normal Kubernetes cluster
-USAGE:
-    install.sh [FLAGS] [OPTIONS]
```

Comment:
we no longer need this scripts.

Reply:
We need to keep this script to help users install Chaos Mesh quickly.

Reply:
Can we add this script back?

Reply:
updated.